packages/toast/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@lidofinance/toast",
+  "version": "0.0.0-semantic-release",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "license": "MIT",
+  "homepage": "https://github.com/lidofinance/ui",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lidofinance/ui.git",
+    "directory": "packages/toast"
+  },
+  "bugs": {
+    "url": "https://github.com/lidofinance/ui/issues"
+  },
+  "keywords": [
+    "lido",
+    "lidofinance",
+    "styled-components",
+    "component",
+    "react-component",
+    "react"
+  ],
+  "dependencies": {
+    "@lidofinance/accordion": "workspace:*",
+    "@lidofinance/button": "workspace:*",
+    "@lidofinance/loaders": "workspace:*",
+    "@lidofinance/styled-system": "workspace:*",
```

Comment:
This package is not used

Reply:
it is using by packages/toast/src/toast-pending/ToastPending.tsx

```
...
export function ToastPending(...
```

Reply:
I meant the `@lidofinance/styled-system` package in dependencies. It adds extra style props for components if you wrap them with HOC `withStyledSystem`.

It seems the package was a copy of the `@lidofinance/box`, that why `@lidofinance/styled-system` is in the dependencies.

Reply:
fixed